hive-llap/test_llap.py

```diff
@@ -5,12 +5,15 @@
 
 from integration_tests.dataproc_test_case import DataprocTestCase
 
-
 class LLAPTestCase(DataprocTestCase):
     COMPONENT = 'hive-llap'
     INIT_ACTIONS = ['hive-llap/llap.sh']
     TEST_SCRIPT_FILE_NAME = 'run_hive_commands.py'
+    ##llap requires zookeeper
     OPTIONAL_COMPONENTS = ["ZOOKEEPER"]
+    ##need initaction repo bucket and the number of llap ndoes to deploy
+    METADATA="num-llap-nodes=1,init-actions-repo=gs://jtaras-init-actions2"
```

Comment:
Should there be a separate test where num-llap-nodes is not passed and default value is used?

Reply:
fixed